lang/cs/Org.Apache.REEF.Client/Org.Apache.REEF.Client.csproj

```diff
@@ -72,12 +88,23 @@ under the License.
       <Project>{75503f90-7b82-4762-9997-94b5c68f15db}</Project>
       <Name>Org.Apache.REEF.Examples</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Org.Apache.REEF.Bridge.JAR\Org.Apache.REEF.Bridge.JAR.csproj">
```

Comment:
This reference causes a problem when building within Visual Studio. 
Similiar to O.A.R.tests.csproj, consider adding:
  <!--begin jar reference-->
  <PropertyGroup>
    <AfterBuildDependsOn>
      $(AfterBuildDependsOn);
      CopyJarFiles;
    </AfterBuildDependsOn>
  </PropertyGroup>
  <Target Name="AfterBuild" DependsOnTargets="$(AfterBuildDependsOn);" />
  <!--end jar reference-->